index.html

```diff
@@ -4744,6 +4725,23 @@ <h2>Verification Method Revocation</h2>
     </section>
 
 
+    <p class="note">
+In order to verify a signature of proof for a verification method which has
+been revoked in a trustless system a DID meethod needs to support either
+`versionId` or `versionTime`, as well as the `updated` and `nextUpdate` DID
+document metadata properties. A verifier can validate a signature or proof of a
+revoked key if and only if all of the following are true:
+    <br/><br/>
+The proof or signature contains the `versionId` or `versionTime` to refer to
+the DID document that was used at the point the signature or proof was created.
+    <br/>
+The verifier can determine the point in time at which the signature or proof
+was made, e.g., it was anchored on a blockchain.
+    <br/>
+For the resolved DID document metadata, the `updated` timestamp is before, and
+the `nextUpdate` timestamp is after, the point in time at which the signature or
+proof was made.
+    </p>
```

Comment:
This includes textual (but I think non-substantive) changes, as well as changes to the formatting markup (multiple paragraphs without indentation do not communicate the list that is intended).
```suggestion
    <div class="note">
      <p>
To verify a signature of proof for a verification method which has been revoked
in a trustless system, a DID method needs to support either or both of the
`versionId` or `versionTime`, as well as both the `updated` and `nextUpdate`,
DID document metadata properties. A verifier can validate a signature or proof
of a revoked key if and only if all of the following are true:
      </p>
      <ul>
        <li>
The proof or signature includes the `versionId` or `versionTime` of the DID
document that was used at the point the signature or proof was created.
        </li>
        <li>
The verifier can determine the point in time at which the signature or proof
was made, e.g., it was anchored on a blockchain.
        </li>
        <li>
For the resolved DID document metadata, the `updated` timestamp is before, and
the `nextUpdate` timestamp is after, the point in time at which the signature or
proof was made.
        </li>
      </ul>
    </div>
```

Reply:
I am content with the language as-is, now that it has been shifted, with one caveat: I think we need to define "trustless." Here, I believe it means a system where no additional metadata is available that a verifier considers trustworthy.